From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@PawanAgarwal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@spelluru 
FYI

carolinacmoravia committed:
CC103367: Adding potential missing content

Hello, @HannibalSII,

This proposed file change comes from https://github.com/MicrosoftDocs/azure-docs.it-it/pull/64. Could you review this and help to merge if agreed?

Many thanks in advance.

carolinacmoravia committed:
C9757: Incorrect hyperlink structure 

Hello, @sasubram, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version. The structrure of the link is incorrect.  



Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
C9740: Different format in target version due to missing spaces 

Hello, @mahesh-unnikrishnan,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  



Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

carolinacmoravia committed:
C9805: Asterisk issue 

Hello, @andkjell, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  An escape character must be added before asterisk at line 33.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

curtand committed:
compare member and guest user role default permissions

curtand committed:
Acrolinx changes

curtand committed:
link fix

curtand committed:
link fix

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vince0118

curtand committed:
owners and other links fixes

curtand committed:
owners and other links fixes 02

curtand committed:
clean up object ownership organisation

curtand committed:
clean up object ownership organisation 02

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vince0118

Ja-Dunn committed:
edit pass: two-iot-edge-articles

carolinacmoravia committed:
L10866: Corrupted characters �

Hello, @rwike77,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.
We need to fix the corrupted characters.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

Ja-Dunn committed:
edit pass: two-iot-edge-articles

Ja-Dunn committed:
edit pass: two-iot-edge-articles

barclayn committed:
adding updated security center with SIEM article

Ja-Dunn committed:
edit pass: two-iot-edge-articles

barclayn committed:
adjusting article to meet acrolynx requirements

barclayn committed:
adding new article to TOC

barclayn committed:
trying to get a link to a section

barclayn committed:
removing reference to a specific h3 in another article

barclayn committed:
making some of John's suggested changes

barclayn committed:
tweaks

barclayn committed:
trying to fix the link to the section

Ja-Dunn committed:
edit pass: two-iot-edge-articles

Ja-Dunn committed:
edit pass: two-iot-edge-articles

kanshiG committed:
Update change-feed.md

Added details for the Change feed

cephalin committed:
add win+python+postgres article

cephalin committed:
acrolinx&validation

Ja-Dunn committed:
edit pass: two-iot-edge-articles

Commit:
Added release notes for version 1.20

Commit:
Update local-emulator.md

Ja-Dunn committed:
edit pass: two-iot-edge-articles

Commit:
Update local-emulator.md

Commit:
added kb live encoder doc

Commit:
acrolinx fixes

curtand committed:
add links to Flow and Power Apps docs

curtand committed:
convert to Note

curtand committed:
fix Note

curtand committed:
fix link intro

spelluru committed:
schedule start and stop of Azure SSIS IR

spelluru committed:
updated

cephalin committed:
bring up to current MVC guidance

cephalin committed:
metadata

cephalin committed:
acrolinx

cephalin committed:
update repo info

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vince0118

curtand committed:
add to ToC

curtand committed:
add Next steps

spelluru committed:
new article - how to schedule Azure SSIS IR

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ssisautomate0125

Commit:
resolved document issues

Commit:
resolved document issues

Commit:
resolved document issues

Commit:
resolved document issues

billmath committed:
updating toc for saas apps

billmath committed:
fixing accidental deletion

billmath committed:
fixing link

divyaswarnkar committed:
commit1

spelluru committed:
Added more steps

billmath committed:
removing old page

rboucher committed:
update TOC to swap monitoring article overviews

rboucher committed:
fix dashes

spelluru committed:
Added monitoring steps

spelluru committed:
updates

wh-alice committed:
fix cli link in b folder and part of c folder

wh-alice committed:
fix cli link in container-service folder

wh-alice committed:
fix cli link in cosmos related folder

wh-alice committed:
fix cli link from d folder to l folder

wh-alice committed:
revert one change in conceptual part

wh-alice committed:
fix cli link in n, o, p folder

wh-alice committed:
fix cli link in storage folder

wh-alice committed:
add one more

wh-alice committed:
apply one more fix

wh-alice committed:
fix cli link in part of s folder and virtual-machine-set* folder

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into viral0125

curtand committed:
change viral to trial

cephalin committed:
add repo info

cephalin committed:
Merge branch 'master' into python

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tocfix

billmath committed:
updating

billmath committed:
updating

Commit:
Update local-emulator.md

barclayn committed:
making changes to comply with reviewers comments

barclayn committed:
fixing the toc

barbkess committed:
buzz fix - SQL DW does not support database-level firewall rules

Ja-Dunn committed:
edit pass: two-iot-edge-articles

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into vince0118

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tocfix

curtand committed:
commented out section intended for Feb 2018

barbkess committed:
acrolinx

PRMerger10 committed:
Merge pull request #30862 from carolinacmoravia/patch-535

C9757: Incorrect hyperlink structure

PRMerger10 committed:
Merge pull request #31213 from carolinacmoravia/patch-539

C9740: Different format in target version due to missing spaces

billmath committed:
fixing toc adding json

Commit:
Merge pull request #31954 from carolinacmoravia/patch-550

L10866: Corrupted characters �

rjagiewich committed:
Merge pull request #32049 from barclayn/asc-siem

adding updated security center with SIEM article

Brenduns committed:
Fixing typos

Ja-Dunn committed:
fix

carolinacmoravia committed:
C8914: Inconsistent format in target version due to incorrect indenta…

Hello, @JYOTHIRMAISURI,

Localization team has reported source content issue that causes localized version to have different format compared to en-us version.
The indentation is wrongly located in the article.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

PRMerger19 committed:
Merge pull request #32354 from Brenduns/daily-fixes-0129

Fixing typos

CarlRabeler committed:
gilad

Brenduns committed:
updateing applicability

carolinacmoravia committed:
L10088: Change comment format to HTML #30668

Hello, @anandyadavMSFT,

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version. Issue: [comment]: <> format is not being processed by MS Transformer. Please, add to allow transformer to block this content

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

PRMerger12 committed:
Merge pull request #32163 from David-Noble-at-work/patch-1

Added release notes for version 1.20, released on January 26, 2018

divyaswarnkar committed:
commit2

spelluru committed:
fixed a couple of issues

solankisamir committed:
Fix example

CarlRabeler committed:
acro

mimig1 committed:
Cosmos DB: Updated and moved note about container support

PRMerger10 committed:
Merge pull request #32358 from Brenduns/daily-fixes-0129

updateing applicability

PRMerger18 committed:
Merge pull request #32357 from carolinacmoravia/patch-581

L10088: Change comment format to HTML #30668

PRMerger15 committed:
Merge pull request #32355 from carolinacmoravia/patch-580

C8914: Inconsistent format in target version due to incorrect indenta…

PRMerger18 committed:
Merge pull request #32362 from solankisamir/policy_edit

Fix example

mimig1 committed:
Cosmos DB: Corrected personalization use case image

mimig1 committed:
made smaller

GitHubber17 committed:
Merge pull request #31737 from curtand/vince0118

compare member and guest user role default permissions

Commit:
Fixed typo

v-thepet committed:
fix broken links

spelluru committed:
fixed the broken link

ShawnJackson committed:
Update how-to-vscode-debug-csharp-module.md

cherylmc committed:
Update point-to-site-vpn-client-configuration-azure-cert.md

rwike77 committed:
updated info

PRMerger20 committed:
Merge pull request #32119 from kanshiG/patch-26

Update change-feed.md

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into viral0125

rwike77 committed:
update

rwike77 committed:
fix

mimig1 committed:
Cosmos DB: Editorial corrections

GitHubber17 committed:
Merge pull request #32270 from billmath/tocfix

updating toc for saas apps

curtand committed:
remove .vscode folder

PRMerger10 committed:
Merge pull request #32363 from mimig1/patch-200

Cosmos DB: Updated and moved note about container support

PRMerger20 committed:
Merge pull request #32367 from v-thepet/linkfix

fix broken links

ShawnJackson committed:
Update how-to-vscode-develop-csharp-module.md

Commit:
Updated SQL RPs

v-shils committed:
Merge pull request #32278 from divyaswarnkar/liquid

commit1

Commit:
Updated DNS entries

PRMerger16 committed:
Merge pull request #32369 from mimig1/patch-201

Cosmos DB: Editorial corrections

PRMerger14 committed:
Merge pull request #32214 from curtand/viral0125

add links to Flow and Power Apps docs

CarlRabeler committed:
removed dup

mimig1 committed:
Cosmos DB: Clarified API level support in logging.md

CarlRabeler committed:
acro

PRMerger16 committed:
Merge pull request #32371 from rwike77/freshness

Freshness

ShawnJackson committed:
Merge pull request #32165 from Ja-Dunn/two-iot-edge-articles

edit pass: two-iot-edge-articles

v-shils committed:
Merge pull request #32286 from rboucher/monitorTOC

update TOC to swap monitoring article overviews

Commit:
Final typo fixes

Commit:
Merge pull request #32372 from mimig1/patch-202

Cosmos DB: Clarified API level support in logging.md

ecfan committed:
Update schema link

ecfan committed:
Fix typo

PRMerger18 committed:
Merge pull request #32368 from cherylmc/patch-1

Update point-to-site-vpn-client-configuration-azure-cert.md

v-shils committed:
Merge pull request #32365 from mimig1/01292018image

Cosmos DB: Corrected personalization use case image

Commit:
Final typo fixes for HTTPS

PRMerger12 committed:
Merge pull request #32366 from jeffgilb/jeff1301

Fixed typo

PRMerger19 committed:
Merge pull request #32374 from ecfan/fix-schema

Fix schema link

curtand committed:
note added for CLI 2.0

spelluru committed:
added to TOC. fixed the NOTE issue

Ja-Dunn committed:
Merge pull request #32350 from barbkess/buzz

buzz fix - SQL DW does not support database-level firewall rules

rwike77 committed:
changed author

bandersmsft committed:
cost mgt-adding Activate subs and accounts

PRMerger14 committed:
Merge pull request #32300 from wh-alice/tfolder

fix cli link in part of s folder and virtual-machine-set* folder

PRMerger20 committed:
Merge pull request #32299 from wh-alice/rstfolder

Fix cli link in storage folder

Ja-Dunn committed:
Merge pull request #32359 from CarlRabeler/gilad

gilad

PRMerger16 committed:
Merge pull request #32297 from wh-alice/dfolder

fix cli link from d folder to l folder

PRMerger14 committed:
Merge pull request #32295 from wh-alice/cfolder2

fix cli link in cosmos related folder

PRMerger12 committed:
Merge pull request #32294 from wh-alice/cfolder

fix cli link in container-service folder

ktoliver committed:
Merge pull request #32200 from dbgeorge/clipper

added kb live encoder doc

PRMerger14 committed:
Merge pull request #32293 from wh-alice/bfolder

fix cli link in b folder and part of c folder

rwike77 committed:
updated toc

Commit:
Merge pull request #32385 from rwike77/remove

changed author

Ja-Dunn committed:
Merge pull request #32298 from wh-alice/nopfolder

fix cli link in n, o, p folder

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into remove

Ja-Dunn committed:
Merge pull request #32221 from spelluru/ssisautomate0125

schedule start and stop of Azure SSIS IR

rwike77 committed:
updated links

rwike77 committed:
redirect

bandersmsft committed:
fixed image alignment

Ja-Dunn committed:
Merge pull request #32386 from bandersmsft/master

cost mgt-adding Activate subs and accounts

cynthn committed:
Update quick-create-powershell.md

ktoliver committed:
Merge pull request #32143 from cephalin/python

flask+db article

cynthn committed:
Update tutorial-iis-sql.md

cynthn committed:
Update tutorial-iis-sql.md

Ja-Dunn committed:
Merge pull request #32387 from rwike77/remove

Remove

PRMerger10 committed:
Merge pull request #31288 from carolinacmoravia/patch-541

C9805: Asterisk issue

rwike77 committed:
freshness

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger15 committed:
Merge pull request #27001 from carolinacmoravia/patch-423

CC103367: Adding potential missing content

PRMerger20 committed:
Merge pull request #32393 from cynthn/patch-58

Update quick-create-powershell.md

v-alje committed:
Merge pull request #32399 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #32394 from cynthn/patch-59

Update tutorial-iis-sql.md

Commit:
Merge pull request #32398 from rwike77/patch-22

freshness

PRMerger20 committed:
Merge pull request #32384 from curtand/rbac0129

note added for CLI 2.0
